docs/sources/reference/components/prometheus.exporter.catchpoint.md

```diff
@@ -0,0 +1,109 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/prometheus.exporter.catchpoint/
+description: Learn about prometheus.exporter.catchpoint
+title: prometheus.exporter.catchpoint
+---
+
+# prometheus.exporter.catchpoint
+
+The `prometheus.exporter.catchpoint` component embeds
+[catchpoint_exporter](https://github.com/grafana/catchpoint-prometheus-exporter) for collecting statistics from a Catchpoint account.
```

Comment:
```suggestion
The `prometheus.exporter.catchpoint` component embeds the [catchpoint_exporter][] for collecting statistics from a Catchpoint account.
```

What is the Catchpoint exporter embedded into? This sentence needs a little refactoring after this question is answered.

Reply:
This question hasn't been answered and the suggestion hasn't been accepted/rejected. 

I feel like "embeds" isn't the right term here.  Maybe its better to state:

```
The `prometheus.exporter.catchpoint` component uses the [catchpoint_exporter][] to collect statistics from a Catchpoint account.
```

Is this still correct?